skyportal/handlers/api/source.py

```diff
@@ -1004,49 +1054,6 @@ def get_sources(
                                 break
                 if not passes_filter:
                     continue
-            if (
```

Comment:
I'm happy to see this go into the direct query of the DB. 

We should think about making a partial text search of the comments at some point. I'm sure it is something that's pretty commonly done in similar situations.

Reply:
Yeah definitely, just not obvious how to do easily.